Engine/Features/GlowEsp.cpp

```diff
@@ -88,48 +88,48 @@ void cGlowEsp::applyGlow() {
             }
         }
         
-        for(int i = 0; i < weaponCounter; i++) {
-            if(registeredEntitys[i] != 0x0) {
-                cEntityManager* entity = engine->DefineGlowEntity(registeredEntitys[i]);
-                if(entity->isValidGlowEntity()) {
-
-                    sGlowEntity glowObject = entity->GetGlowObject();
-                    
-                    bool isWeapon = entity->isWeapon();
-                    bool isBomb = entity->isBomb();
-                    
-                    float alpha = settings->GetGlowalpha();
-                    std::string colorBase;
-                    
-                    if((isWeapon || isBomb) && glowObject.RenderWhenOccluded) {
-                        continue;
-                    }
-                    
-                    if (isWeapon) {
-                        colorBase = "255,0,0";
-                    } else if (isBomb) {
-                        colorBase = "0,255,0";
-                    } else {
-                        delete entity;
-                        continue;
-                    }
-                    
-                    std::vector<std::string> colors = settings->split<std::string>(colorBase, ",");
-                    float red = atof(colors[0].c_str()) / 255;
-                    float green = atof(colors[1].c_str()) / 255;
-                    float blue = atof(colors[2].c_str()) / 255;
-                    
-                    glowObject.r = red;
-                    glowObject.g = green;
-                    glowObject.b = blue;
-                    glowObject.a = alpha;
-                    glowObject.RenderWhenOccluded = true;
-                    glowObject.RenderWhenUnoccluded = false;
-                    
-                    entity->setGlow(glowObject);
-                }
-            }
-        }
+//        for(int i = 0; i < weaponCounter; i++) {
```

Comment:
Commented code is dead code.
Remove it please. Therefor it is a version control.